
From: Kaigai Kohei <kaigai@ak.jp.nec.com>

This patch declares atomic_inc_return() as the alias of atomic_add_return()
and atomic_dec_return() as an alias of atomic_dec_return().

Signed-off-by: KaiGai, Kohei <kaigai@ak.jp.nec.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-arm/atomic.h |    2 ++
 1 files changed, 2 insertions(+)

diff -puN include/asm-arm/atomic.h~atomic_inc_return-for-arm include/asm-arm/atomic.h
--- 25/include/asm-arm/atomic.h~atomic_inc_return-for-arm	2004-09-11 16:08:22.667656832 -0700
+++ 25-akpm/include/asm-arm/atomic.h	2004-09-11 16:08:22.670656376 -0700
@@ -149,6 +149,8 @@ static inline void atomic_clear_mask(uns
 
 #define atomic_inc_and_test(v)	(atomic_add_return(1, v) == 0)
 #define atomic_dec_and_test(v)	(atomic_sub_return(1, v) == 0)
+#define atomic_inc_return(v)    (atomic_add_return(1, v))
+#define atomic_dec_return(v)    (atomic_sub_return(1, v))
 
 #define atomic_add_negative(i,v) (atomic_add_return(i, v) < 0)
 
_
